src/components/Header/Header.jsx

```diff
@@ -183,6 +189,33 @@ export function Header(props) {
     };
   }, [user.userid, props.auth.firstName]);
 
+  useEffect(() => {
+    console.log("Current Profile Pic:", profilePic);
+  }, [profilePic]);
+
+  // useEffect(() => {
+  //   const handleResize = () => {
+  //     console.log('Window size: ', window.innerWidth);
+  //   };
+  
+  //   // Add event listener
+  //   window.addEventListener('resize', handleResize);
+  //   handleResize();
+  
+  //   return () => {
+  //     window.removeEventListener('resize', handleResize);
+  //   };
+  // }, []); // Empty dependency array means this effect will only run once, similar to componentDidMount
+
```

Comment:
I recommend removing commented-out code. GitHub has a history of all changes, so old versions are always accessible. Removing it makes the code cleaner and more readable.